docs/source/ContributionGuide.rst

```diff
@@ -242,6 +242,38 @@ approved by your employer) and you grant a full, complete, irrevocable
 copyright license to all users and developers of the Avocado project,
 present and future, pursuant to the license of the project.
 
+Code Review
+-----------
+
+Every single Pull Request in Avocado has to be reviewed by at least one
+other developer. All members of the core team have permission to merge
+a Pull Request, but there are some conditions that have to be fulfilled
+before merging the code:
+
+- Pull Request has to pass the CI tests.
+- One 'Approved' code review should be given.
+- No explicit disapproval should be present.
```

Comment:
Well is it `No` or is it that the ratio has to be +1? Because we already merged PRs with 1 NACK and 2 ACKS. Anyway this is not that important and can be tweaked later...